submarine-sdk/pysubmarine/example/deepFM.json

```diff
@@ -0,0 +1,31 @@
+{
+  "input": {
+    "train_data": "./data/tr.libsvm",
+    "valid_data": "./data/va.libsvm",
+    "test_data": "./data/te.libsvm",
+    "type": "libsvm"
+  },
+  "output": {
+    "save_model_dir": "./experiment",
+    "metric": "auc"
+  },
+  "training": {
+    "batch_size" : 512,
+    "field_size": 39,
+    "num_epochs": 3,
+    "feature_size": 117581,
+    "embedding_size": 256,
+    "learning_rate": 0.0005,
+    "batch_norm_decay": 0.9,
+    "l2_reg": 0.0001,
+    "deep_layers": "400,400,400",
+    "dropout": "0.3,0.3,0.3",
+    "batch_norm": false,
+    "optimizer": "adam",
+    "log_steps": 10,
+    "num_threads": 4,
+    "num_gpu": 0,
+    "seed": 77,
+    "mode": "local"
```

Comment:
> Can we reorg the `training` parameters into two parts: `algorithm` and `system`. The `algorithm` part should be more friendly to data scientists and machine learning engineers, and the `system` part would be more advanced.

Agree, it will be more user-friendly for users, I will update it.